File: OTROS/ServerSpace/src/main/java/Business/Ubigeo.java
package Business;

import DataLayer.UbigeoDB;
import EntityLayer.UbigeoEntity;
import java.util.ArrayList;

public class Ubigeo {

    public ArrayList<UbigeoEntity> GetAllItems() {
        UbigeoDB BD = new UbigeoDB();
        return BD.GetAllItems();
    }

    public ArrayList<UbigeoEntity> GetAllItem(int Id) {
        UbigeoDB BD = new UbigeoDB();
        return BD.GetAllItem(Id);
    }

    public UbigeoEntity Save(UbigeoEntity Item) {
        UbigeoDB BD = new UbigeoDB();
        return BD.Save(Item);
    }

    public Boolean Delete(int Id) {
        UbigeoDB BD = new UbigeoDB();
        return BD.Delete(Id);
    }
    public ArrayList<UbigeoEntity> GetUbigeoLikeItem(String Nombre) {
        UbigeoDB BD = new UbigeoDB();
        return BD.GetUbigeoLikeItem(Nombre);
    }

}

File: Generador/File Generado/Business/Ubigeo.py
from DataLayer.UbigeoDB import *
from EntityLayer.UbigeoEntity import *
from Utilidades.Conexion.configMysql import StartTransaction, EndTransaction, Restore


class Ubigeo:
    def Save(Ent: UbigeoSaveModel):
        try:
            StartTransaction()
            data = UbigeoDB.Save(Ent)
            EndTransaction()
            return data
        except Exception as e:
            Restore()
            print(e)
    
    def GetItems():
        try:
            return UbigeoDB.GetItems()
        except Exception as e:
            print(e)
    
    def GetItem(Id: int):
        try:
            return UbigeoDB.GetItem(Id)
        except Exception as e:
            print(e)
    
    def Delete(Id: int):
        try:
            return UbigeoDB.Delete(Id)
        except Exception as e:
            print(e)
    
